_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -1939,6 +1979,89 @@ void ImplementationAvc::BrcPreEnc(
 
     m_brc.PreEnc(task.m_brcFrameParams, m_tmpVmeData);
 }
+
+mfxStatus ImplementationAvc::SubmitToMctf(DdiTask * pTask, bool isSceneChange, bool doIntraFiltering)
+{
+    MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_API, "VideoVPPHW::SubmitToMctf");
+    //mfxStatus sts = MFX_ERR_NONE;
+
+    pTask->bFrameReady   = false;
+    mfxFrameSurface1
+        * pSurfI         = nullptr;
+    pSurfI               = m_core->GetNativeSurface(pTask->m_yuv);
```

Comment:
this line can be merged with previous one

Reply:
Done